script/src/verify.rs

```diff
@@ -584,6 +959,61 @@ impl<'a, DL: CellDataProvider + HeaderProvider> TransactionScriptsVerifier<'a, D
             Err(ScriptError::validation_failure(&script_group.script, code))
         }
     }
+
+    fn chunk_run(
+        &'a self,
+        script_group: &'a ScriptGroup,
+        max_cycles: Cycle,
+        snap: &Option<Snapshot>,
+    ) -> Result<ChunkState<'a>, ScriptError> {
+        let mut machine = self.build_machine(script_group, max_cycles)?;
+
+        let map_vm_internal_error = |error: VMInternalError| match error {
+            VMInternalError::InvalidCycles => ScriptError::ExceededMaximumCycles(max_cycles),
+            _ => ScriptError::VMInternalError(format!("{:?}", error)),
+        };
+
+        let program = self.extract_script(&script_group.script)?;
+        let bytes = machine
+            .load_program(&program, &[])
```

Comment:
If there is a snapshot, it should not be necessary to load the program

Reply:
No, you are wrong, please check https://github.com/nervosnetwork/ckb-vm/blob/develop/tests/test_resume.rs.